testing/tests/DevExpress.ui/hideToasts.tests.js

```diff
@@ -0,0 +1,76 @@
+import $ from 'jquery';
+import notify from 'ui/notify';
+import fx from 'animation/fx';
+import hideToasts from 'ui/toast/hide_toasts';
+
+QUnit.testStart(function() {
+    const markup =
+        '<div id="container"></div>';
+
+    $('#qunit-fixture').html(markup);
+});
+
+QUnit.module('hideToasts', {
+    beforeEach: function() {
+        fx.off = true;
+        this.clock = sinon.useFakeTimers();
+
```

Comment:
```suggestion
const displayTime = 100;
```

to simplify configs below